### DISCUSSION TITLE
Need some help with article "Porting the Ada Runtime to a new ARM board"

### DISCUSSION THREAD

**Discussion by ALSCode** [AdaCore/bb-runtimes#53] [General]
Hello!
First of all, thanks a lot for such useful article! Unfortunately, i'm at a loss a bit  with paragraph "Check your installation". When i trying _build_rts.py --bsps-only --output=build --prefix=arm-eabi/lib/gnat --link stm32f4_, i get message from translator:
_build_rts.py: error: unrecognized arguments: --bsps-only --prefix=arm-eabi/lib/gnat_. I use Windows 10 64 bit and Python 3.9.

Could you please give me a direction to solve this issue?
Thank you!

**Comment by Fabien-Chouteau** [comment]
Thank you @ALSCode,

Unfortunately the blog post is starting to show its age and some of the things mentioned there have changed since it was published.
I recommend to have a look at the README of this repository.

Regards,
